packages/wonder-blocks-banner/src/components/__docs__/banner.stories.js

```diff
@@ -0,0 +1,355 @@
+            the \`text\` prop.`,
+    },
+};
+
+export const Variants: StoryComponentType = () => (
+    <View style={styles.container}>
+        <View style={styles.variantColumn}>
+            <LabelMedium>Full-width layout</LabelMedium>
+            <View style={styles.spacing}>
+                <Banner text="kind: info" kind="info" layout="full-width" />
+            </View>
+            <View style={styles.spacing}>
+                <Banner
+                    text="kind: success"
+                    kind="success"
+                    layout="full-width"
+                />
+            </View>
+            <View style={styles.spacing}>
+                <Banner
+                    text="kind: warning"
+                    kind="warning"
+                    layout="full-width"
+                />
+            </View>
+            <View style={styles.spacing}>
+                <Banner
+                    text="kind: critical"
+                    kind="critical"
+                    layout="full-width"
+                />
+            </View>
+        </View>
+        <View style={styles.variantColumn}>
+            <LabelMedium>Floating layout</LabelMedium>
+            <View style={styles.spacing}>
+                <Banner text="kind: info" kind="info" layout="floating" />
+            </View>
+            <View style={styles.spacing}>
+                <Banner text="kind: success" kind="success" layout="floating" />
+            </View>
+            <View style={styles.spacing}>
+                <Banner text="kind: warning" kind="warning" layout="floating" />
+            </View>
+            <View style={styles.spacing}>
+                <Banner
+                    text="kind: critical"
+                    kind="critical"
+                    layout="floating"
+                />
+            </View>
+        </View>
+    </View>
+);
+
+Variants.parameters = {
+    docs: {
+        storyDescription: `Banners come with two layouts: full-width (default)
+            and floating. Full-width layout gives the banner square edges,
+            and floating layout gives the banner rounded edges. Banners
+            also come in four different kinds: info (default), success,
+            warning, and critical.`,
+    },
+};
+
+export const DarkBackground: StoryComponentType = () => (
+    <View style={styles.darkBackground}>
+        <View style={styles.spacing}>
+            <Banner text="kind: info" kind="info" layout="floating" />
+        </View>
+        <View style={styles.spacing}>
+            <Banner text="kind: success" kind="success" layout="floating" />
+        </View>
+        <View style={styles.spacing}>
+            <Banner text="kind: warning" kind="warning" layout="floating" />
+        </View>
+        <View style={styles.spacing}>
+            <Banner text="kind: critical" kind="critical" layout="floating" />
+        </View>
+    </View>
+);
+
+DarkBackground.parameters = {
+    docs: {
```

Comment:
**suggestion:** instead of including the `darkBackground` View wrapper, we could use the SB `backgrounds` option:

```suggestion
    backgrounds: {
        default: "darkBlue",
    },
    docs: {
```